prow/config.yaml

```diff
@@ -15351,6 +15360,9 @@ periodics:
       - mountPath: /etc/ssh-key-secret
         name: ssh
         readOnly: true
+      - mountPath: /etc/service-account
+        name: service
+        readOnly: true
```

Comment:
(this is for the regional job was actually using a different service account, need to make a different preset for them)